views/admin/tpl/email/module_support_email.tpl

```diff
@@ -0,0 +1,182 @@
+[{assign var="shop"      value=$oEmailView->getShop()}]
+[{assign var="oViewConf" value=$oEmailView->getViewConfig()}]
+[{assign var="userinfo"  value=$oEmailView->getUser()}]
+[{assign var="header_template" value="`$shopTemplateDir`email/html/header.tpl"}]
+[{assign var="footer_template" value="`$shopTemplateDir`email/html/footer.tpl"}]
+
+[{include file=$header_template title=$email_data.subject}]
+
+<table border="0" width="100%" cellspacing="10" cellpadding="2" bgcolor="#FFFFFF" class="column">
+  <tr>
+    <td class="text-pad" style="white-space:nowrap;">
+      <b style="white-space:nowrap;">[{oxmultilang ident="module_support_email_from"}]</b>
+    </td>
+    <td style="white-space:nowrap;">[{$shop->oxshops__oxname->getRawValue()|oxescape}]</td>
```

Comment:
Why do you need to escape here? Doesn't the `value` field method escape automatically?

Reply:
Not sure about that.

Reply:
It runs it through `htmlspecialchars`: https://github.com/OXID-eSales/oxideshop_ce/blob/master/source/Core/Field.php#L107

Idk if this is enough for your case.